executor/src/witgen/generator.rs

```diff
@@ -319,39 +340,32 @@ where
         } else {
             EvaluationRow::Next
         };
-        let evaluated = self.evaluate(identity, row)?;
+        let evaluated = match self.evaluate(identity, row) {
+            Ok(evaluated) => evaluated,
+            Err(cause) => return Ok(EvalValue::incomplete(cause)),
+        };
```

Comment:
So this is how we interface between evaluation results and processor results. Not ideal but I can't think of a way to return early because we return Ok and not Err.

Reply:
What do you mean by "return early"?

Reply:
Ah ok!